scripts/configuration/testProjects.csv

```diff
@@ -1,4 +1,5 @@
 EndToEnd
+dotnet.Tests
```

Comment:
Not a fan of calling it dotnet.Tests, to me it feels like it will encourage people to put all the dotnet tests under this project instead of following the pattern of having separate test projects for the commands.